src/plugins/roomslist/styles/roomsgroups.scss

```diff
@@ -0,0 +1,14 @@
+.conversejs {
+    #chatrooms {
+        .muc-domain-group-toggle {
+            margin: 0.75em 0 0.25em 0;
+        }
+
+        .muc-domain-group-toggle, .muc-domain-group-toggle .fa {
+            color: var(--groupchats-header-color) !important;
+            &:hover {
+                color: var(--chatroom-head-bg-color-dark) !important;
```

Comment:
Is the `!important` directive really necessary? We try to avoid using it unless absolutely necessary, which is rare.

Reply:
I did it to copy what was done for [`group-toggle`](https://github.com/conversejs/converse.js/blob/master/src/plugins/rosterview/styles/roster.scss#L86) and [`open-rooms-toggle`](https://github.com/conversejs/converse.js/blob/master/src/plugins/muc-views/styles/controlbox.scss#L17), but it does seem to work without it, so I’ll remove it.